x-pack/test/functional/page_objects/observability_log_explorer.ts

```diff
@@ -249,7 +249,7 @@ export function ObservabilityLogExplorerPageObject({
       log.info('queryStringParams');
```

Comment:
Shouldn't we rename `OBSERVABILITY_LOG_EXPLORER_URL_STATE_KEY` to `OBSERVABILITY_LOGS_EXPLORER_URL_STATE_KEY`?